_sass/_utilities.scss

```diff
@@ -86,14 +86,32 @@ a {
 
 .texture-bg-img {
   background: url('../img/texture.png') no-repeat bottom center;
-  background-size: contain;
+  background-size: cover;
```

Comment:
does this change how it renders on index.html (hero section)?

Reply:
Yes. The reason I changed it to `cover` is because on some pages, such as the Donation page, where the header is quite narrow, the texture image doesn't fill the full width of the page and has a slightly visible line.

![Screen Shot 2019-03-14 at 3 17 40 pm](https://user-images.githubusercontent.com/23013276/54331121-93b47400-466c-11e9-9d46-01a8b389a806.png)